lib/deck_scraper.rb

```diff
@@ -40,32 +43,28 @@ def get_decks
 
   def get_decks_htd
     mechanize = Mechanize.new
-    @page = mechanize.get('http://www.hearthstonetopdeck.com/')
-    @decks = @page.links
 
     output_decks = []
-    @decks.each do |deck_page|
-      unless !deck_page.href.nil? && !(deck_page.href.include? "deck.php")
```

Comment:
hearthstonetopdeck changed their routing and so the script wasn't pulling any decks from there